app/frontend/src/components/forms/manage/Subscription.vue

```diff
@@ -0,0 +1,145 @@
+                : $t('trans.subscribeEvent.password')
+            "
+          ></v-text-field>
+        </v-col>
+        <v-col cols="12" sm="3">
+          <v-tooltip bottom>
+            <template #activator="{ on, attrs }">
+              <v-btn
+                color="primary"
+                icon
+                small
+                v-bind="attrs"
+                v-on="on"
+                @click="showHideKey"
+              >
+                <v-icon v-if="showSecret">visibility_off</v-icon>
+                <v-icon v-else>visibility</v-icon>
+              </v-btn>
+            </template>
+            <span v-if="showSecret">
+              {{ $t('trans.subscribeEvent.hideSecret') }}
+            </span>
+            <span v-else>{{ $t('trans.subscribeEvent.showSecret') }}</span>
+          </v-tooltip>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col>
+          <v-btn class="mr-5" color="primary" @click="updateSettings">
+            <span>{{ $t('trans.subscribeEvent.save') }}</span>
+          </v-btn>
+        </v-col>
+      </v-row>
+    </v-form>
+  </v-container>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+import { mapFields } from 'vuex-map-fields';
+
+export default {
+  name: 'Subscription',
+  data() {
+    return {
+      loading: false,
+      showConfirmationDialog: false,
+      showDeleteDialog: false,
+      showSecret: false,
+      valid: false,
+      subscriptionFormValid: false,
+      endpointUrlRules: [
+        (v) => !!v || this.$t('trans.formSettings.validEndpointRequired'),
+        (v) =>
+          (v &&
+            new RegExp(
+              /^(ht|f)tp(s?):\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}([-.\w]*[0-9a-zA-Z])*(:(0-9)*)*(\/?)([-a-zA-Z0-9()@:%_+.~#?&//=]*)/g
+            ).test(v)) ||
+          this.$t('trans.formSettings.validEndpointRequired'),
+      ],
+      endpointTokenRules: [
+        (v) => !!v || this.$t('trans.formSettings.validBearerTokenRequired'),
+      ],
+    };
+  },
+  computed: {
+    ...mapGetters('form', ['apiKey', 'form', 'permissions', 'version']),
+    ...mapFields('form', ['subscriptionData']),
+  },
+  methods: {
+    ...mapActions('form', ['updateSubscription', 'readFormSubscriptionData']),
+    ...mapActions('notifications', ['addNotification']),
+    async updateSettings() {
+      try {
+        if (this.$refs.subscriptionForm.validate()) {
+          let subscriptionData = {
+            ...this.subscriptionData,
+            formId: this.form.id,
+          };
+          await this.updateSubscription({
+            formId: this.form.id,
+            subscriptionData: subscriptionData,
+          });
+
+          this.readFormSubscriptionData(this.form.id);
+        }
+      } catch (error) {
+        this.addNotification({
+          message:
+            'An error occurred while attempting to update the settings for this form.',
+          consoleError: `Error updating settings for ${this.form.id}: ${error}`,
```

Comment:
I think this `message` and `consoleError` might have been missed when setting up the translations?